header/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
doctype is case insensitive (https://html.spec.whatwg.org/multipage/syntax.html#the-doctype) I don't see the reason for this change to be in this PR. Google uses all lowercase.

Reply:
i usually use lowercase too...guess one of my linters was overzealous and autoformatted this?